kdbg/xsldbgdriver.cpp

```diff
@@ -1309,6 +1319,12 @@ XsldbgDriver::parseInfoLine(const char */*output*/, QString & /*addrFrom*/,
     return false;
 }
 
+QString XsldbgDriver::parseInfoTarget(const char* output)
+{
+    Q_UNUSED(output);
+    return QString();
+}
+
```

Comment:
Please follow the precedent of other `parse` functions, which have `/*output*/` in comment and don't use `Q_UNUSED`.